scripts/core/auth/login-modal.html

```diff
@@ -23,12 +23,19 @@
                         </div>
                     </button>
                 </fieldset>
+
+                <fieldset class="buttons" ng-show="allowedAuth.google">
+                    <button class="btn btn--sd-green"
+                        ng-click="openLoginPopup('google')" translate>Log In with Google</button>
```

Comment:
There was a comment from Vince about the consistency of this label. What if we use `{{ 'Log In' | translate }} with Google` as above for now?